components/net/tests/fetch.rs

```diff
@@ -9,6 +9,7 @@ use crate::{
     create_embedder_proxy, fetch, make_server, make_ssl_server, new_fetch_context,
     DEFAULT_USER_AGENT,
 };
+use crossbeam_channel::{channel, Sender};
```

Comment:
This function is called `unbounded`, not `channel`. The same error appears in some other files, which is why CI is currently failing.

Reply:
Oh woops, didn't think about unit tests